is_core/menu.py

```diff
@@ -51,14 +57,28 @@ def get_menu_items(self, items):
                 if item.submenu_items:
                     item = deepcopy(item)
                     item.submenu_items = list(self.get_menu_items(item.submenu_items))
+
                 yield item
+
             else:
-                item = self.site._registry[item]
-                if isinstance(item, UIISCore):
-                    menu_item = item.get_menu_item(self.request, group)
+                core = self.site._registry[item]
+
+                try:
+                    menu_item = self.get_menu_item(
+                        core.verbose_name_plural,
+                        core.menu_url(self.request),
+                        core.menu_group,
+                    )
+
                     if menu_item:
-                        yield item
+                        yield menu_item
+
+                except NotImplementedError:
```

Comment:
v tom nevidím problém ten bych naopak předával, také bych tam nedával args a kwargs (volající má stejně pevně daný seznam parametrů), jen bych tam předal to core ať si z něj vytáhne co chce